src/data/items.h

```diff
@@ -1359,6 +1359,7 @@ const struct Item gItems[] =
         .name = _("Rare Candy"),
         .itemId = ITEM_RARE_CANDY,
         .price = 10000,
+        .holdEffectParam = 0,
```

Comment:
Is there a point in this? The default value of an item's field is 0 and it remains 0 if it's not given a different value, so... :eyes: